src/app/shared/components/users-list/users-list.component.html

```diff
@@ -122,9 +122,12 @@
           </div>
   
           <mat-menu #actions="matMenu" class="header_menu">
-            <button mat-menu-item (click)="onBlock(user)">
+            <button *ngIf="user.status !== 'Blocked'" mat-menu-item (click)="onBlock(user)">
               Заблокувати користувача
             </button>
+            <button *ngIf="user.status == 'Blocked'" mat-menu-item (click)="unBlock(user)">
+              Розблокувати користувача
+            </button>
```

Comment:
consider applying another approach, there is a lot of duplication. You can use only one button and change the only text displayed. 
1. you can create only one method Block/Unblock and pass there user, and directly in the method to handle the action because you know user.status